.cirrus.yml

```diff
@@ -784,3 +785,18 @@ static_build_task:
 
     binaries_artifacts:
         path: "result/bin/podman"
+
+
+darwin_build_task:
+    depends_on:
+        - "gating"
+    osx_instance:
+        image: catalina-base
+    setup-script:
+        - brew install go
+        - brew install go-md2man
+    build-script:
+        - make podman-remote-darwin
```

Comment:
we could, that will spit out a zip file which is easier to consume.  I'm fine either way, since we'll likely need to revisit/revise this anyway, soon as @ashley-cui figures out the signing stuff :smile: